dashboard/settings.py

```diff
@@ -417,16 +415,8 @@
         'givenName': ('first_name', ),
         'sn': ('last_name', ),
     }
-else:
-    AUTHENTICATION_BACKENDS += ('django.contrib.auth.backends.ModelBackend',)
-    LOGIN_REDIRECT_URL = '/'
-    LOGOUT_REDIRECT_URL='/'
-
-# Give an opportunity to disable LTI
-if ENV.get('STUDENT_DASHBOARD_LTI', False):
-    if not 'django.contrib.auth.backends.ModelBackend' in AUTHENTICATION_BACKENDS:
-        AUTHENTICATION_BACKENDS += ('django.contrib.auth.backends.ModelBackend',)
```

Comment:
I guess this was always there from before since LTIV1p1, some how the logic here adding it exclusively. I did not give any serious though on this when upgrading to v1p3. Thanks for taking care of this